src/lib/components/HoverCard/portal.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang='ts' context='module'>
+    type PortalProps = ComponentProps<InstanceType<typeof Portal>>;
+    export type HoverCardPortalProps = {
+        container?: PortalProps['target'];
+    };
```

Comment:
Since we're using $$restProps:

```suggestion
    type PortalProps = ComponentProps<InstanceType<typeof Portal>>;
    export type HoverCardPortalProps = Omit<PortalProps, 'target'> & {
    	container?: PortalProps['target'];
    }:
```